django_select2/widgets.py

```diff
@@ -210,12 +210,13 @@ def render(self, name, value, attrs=None, choices=()):
 
         return mark_safe(s)
 
-    class Media:
+    def _get_media(self):
+        media = forms.Media()
         if RENDER_SELECT2_STATICS:
-            js = get_select2_js_libs()
-            css = {
-                'screen': get_select2_css_libs(light=True),
-            }
+            media._js = get_select2_js_libs()
```

Comment:
Please create an instance of `forms.Media` with the right arguments rather then accessing private attribute on the instance.